server/views/components/nav/macro.njk

```diff
@@ -0,0 +1,62 @@
+{% macro govukNav(user, userSvg, isTranslationsEnabled, currentLng) %}
+  {% if user %}
+    <nav aria-label="Launchpad Home Account Navigation">
+      <div class="launchpad-home-header__navigation">
+        <h2 id="launchpad-home-header__navigation-heading" class="govuk-visually-hidden">
+          Launchpad Home Navigation Menu
+        </h2>
+
+        <div class="launchpad-home-header__navigation__items">
+          {{ translationsNav(isTranslationsEnabled, currentLng) }}
+          {{ userMenu(user, userSvg) }}
+        </div>
+      </div>
+    </nav>
+  {% endif %}
+{% endmacro %}
+
+{% macro translationsNav(isTranslationsEnabled, currentLng) %}
+  {% if isTranslationsEnabled %}
+    <div class="launchpad-home-header__navigation__item launchpad-home-header__navigation__item__translations">
+      {% set translations = [
```

Comment:
Could possibly move this list into the route or somewhere and drag through from config 🤔 Saves config being inside a nunjucks template?

Reply:
Good idea 😄